tools/pd-simulator/simulator/client.go

```diff
@@ -50,7 +51,7 @@ type Client interface {
 }
 
 const (
-	pdTimeout             = time.Second
+	pdTimeout             = 3 * time.Second
```

Comment:
Why change it?

Reply:
to keep consistent with https://github.com/tikv/pd/blob/8215f3088c7373512909d60c9d0ae7d9a577091b/client/option.go#L27